src/pipx/venv.py

```diff
@@ -77,9 +77,9 @@ def get_venv_dir(self, package_name: str) -> Path:
         """Return the expected venv path for given `package_name`."""
         return self._root.joinpath(canonicalize_name(package_name))
 
-    def verify_shared_libs(self) -> None:
+    def verify_shared_libs(self, pip_args: List[str], verbose: bool) -> None:
```

Comment:
I don't really understand the purpose of this function, BTW.

```suggestion
    def verify_shared_libs(self, pip_args: Optional[List[str]] = None, verbose: bool) -> None:
```

Reply:
a small grep tells me that it's not used anywhere - should I get rid of it ?

Reply:
Though I like the `--skip-maintenance` flag. I'm not sure we should always refrain from upgrading the libs when listing packages.

Reply:
In that case should we provide a `--pip-args` there too ? 

Otherwise, if I understand correctly, that means that `pipx list` can arbitrarily connect to `pypi`, and user would not be able to provide a custom index_url nor certificate.

Reply:
> In that case should we provide a `--pip-args` there too ?

Yes, that would be the logical consequence.